src/channels/03-music.yml

```
@@ -184,6 +184,8 @@ schedule:
     - "06:00":
     - "06:30":
     - "07:00":
+      title: Moss Garden
+      url: https://moss.garden/
```

```suggestion
        title: Moss Garden
        url: https://moss.garden/
```
Yaml is whitespace-sensitive, so these lines need to be indented for the site to build

---

Do you want to add an author name?

---

hi Eva! it's a handmade internet radio that plays ambient and environmental music. I'm working on a generative piece that takes listener-submitted field recordings that gradually flow into and intersperse with the songs.

---

Nice! I like it

You'll need to update the yaml file before I can approve and merge. Let me know if you need help